openy_gated_content.module

```diff
@@ -173,3 +210,47 @@ function openy_gated_content_user_pass_validation(array &$form, FormStateInterfa
     }
   }
 }
+
+/**
+ * Implements hook_entity_base_field_info().
+ */
+function openy_gated_content_entity_base_field_info(EntityTypeInterface $entity_type) {
+
+  $allowed_entity_types = [
```

Comment:
Can we use `$gcConfig = \Drupal::config('openy_gated_config.settings')` here instead of hardcoded array?